lib/PrintAsObjC/PrintAsObjC.cpp

```diff
@@ -119,6 +119,25 @@ static StringRef getNameForObjC(const ValueDecl *VD,
   return VD->getName().str();
 }
 
+static Optional<StringRef>
+printSwiftEnumElemNameInObjC(const EnumElementDecl *EL, llvm::raw_ostream &OS,
```

Comment:
Stepping back, the fact that I can't figure out what the function is doing is not a great sign. :-/